src/EventGridEmulator.Tests/EventGridEmulator.Tests.csproj

```diff
@@ -5,6 +5,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="6.11.0" />
```

Comment:
I think for uniformity I'd keep it without fluent unless we decide to switch to fluent in all our projects. Similar to how we decided to stick to FakeItEasy instead of Moq (before the whole Moq scandal)